### ISSUE TITLE
A11y_WindowsUILibraryv2.4_Tab View_Tab View with close button_Keyboard: Closed Tab button is not in keyboard focus order.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/microsoft-ui-xaml#3150]
["Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11y_WindowsUILibraryv2.4;#A11yTCS;#A11yMAS;#DesktopApp;#WCAG1.3.1;#WCAG2.1.1;#A11ySev2;#Windows UI Library;#STP;#NVDA;#Win32;#Narrator;#FTP;#Benchmark;#A11ySev2

**Environment Details:**
Application Name: Windows UI Library v2.4
Version: Version 2.4
OS: Windows10

**Additional Details:**
MAS References:WCAG2.1.1,WCAG1.3.1

**Repro Steps:**

1.	Install and open theWindows UI Library v2.4 application.
2.	Navigate to Menu pane on the left side of the screen.
3.	Navigate to "Updated Controls" List Item and click Tab View->Navigate to right side of the pane.
4.	Navigate to down side to "Tab view with close buttons".
5.	Navigate to tab and then close tab button.

**Actual:**

Closed Tab button is not in keyboard focus order. After pressing Tab key focus move to other tab and also we are not able to perform action on this buttons when screen reader (Narrator)  is On.

**Same issue is also observed in "Text Box Family" screen in which focus is not move to "Clear All"  and "Show Password" button.**

**Expected:**

Closed Tab button and Clear all buttons should be in keyboard focus order for closing tab.

**User Impact:**

Keyboard users will not be able to closed tab with the help of keyboard key.

**Attachment link for Reference:**
[#3150.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5088790/3150.zip)

**Comment by Felix-Dev** [comment]
See https://github.com/microsoft/microsoft-ui-xaml/issues/1648:

> It's intentional that the close button isn't reachable with keyboard (same as Edge/Chrome).

**Comment by YuliKl** [comment]
Following up with accessibility team to determine the best resolution for this issue. Expecting recommendation by 8/27.

**Comment by YuliKl** [comment]
Following up with the Edge team to understand their motivation for close buttons' focus behavior.

**Comment by YuliKl** [comment]
Resolving this issue as by design. Accessibility team suggested that we should avoid polluting keyboard focus order with extra focus targets. Users are able to close the selected tab using the Ctrl+F4 shortcut. This does imply that there's no built-in way for users to close the focused not-selected tab, but that limitation was deemed acceptable.
